epytope/tutorials/PolymorphicEpitopePrediction.ipynb

```diff
@@ -304,7 +304,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.0"
+   "version": "3.6.13"
```

Comment:
Not sure tbh. I think it does not need to be changed. I'll change it back

Reply:
Looks like it is still the same Python version

Reply:
Thats weird because it is not the case on my local and it doesn't appear in changed files via `git status`. Let's leave it like this since it should run on python `3.6` and `3.7` anyhow?